### PR TITLE
SDK changes

### PR DESCRIPTION
**Pull request by JeeveshJ7**

### CODE REVIEW

File: sdk/SmartUI_SDK.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    browserName: "chrome",
+    version: "latest",
+    "LT:Options": {
+		"username": USERNAME,
+		"accessKey": KEY,
+		"project": "<PROJECT_NAME>",
+		"w3c": true,
+        name: "<TEST_NAME>", // name of the test
+        build: "<BUILD_NAME", // name of the build
+        visual: true,
```
Comment (sushobhit-lt) [inline sdk/SmartUI_SDK.js:20]: we can skip the "LT:Options"

File: package.json [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
     "minimatch": "^3.0.5",
     "moment": "^2.29.4",
+    "playwright": "^1.35.1",
     "scroll-to-bottomjs": "^1.1.0",
-    "selenium-webdriver": "^3.6.0"
+    "@lambdatest/selenium-driver": "^1.0.0",
+    "@lambdatest/smartui-cli": "^2.0.0",
+    "selenium-webdriver": "^4.16.0"
   },
+  "type": "module",
```
Comment (sushobhit-lt) [inline package.json:11]: Is there any specific need, to make this complete project as module?

Comment (JeeveshJ7) [reply package.json:11]: @pinanks has specifically asked to add the same.

File: sdk/SmartUI_SDK_local.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import  { Builder, By, Key, until } from 'selenium-webdriver';
+import { smartuiSnapshot } from '@lambdatest/selenium-driver';
+
+(async function example() {
+    let driver = await new Builder()
+        .forBrowser('edge')
```
Comment (sushobhit-lt) [inline sdk/SmartUI_SDK_local.js:6]: Can we use same browser for local and cloud ?

Comment (JeeveshJ7) [reply sdk/SmartUI_SDK_local.js]: done

**Review by sushobhit-lt — APPROVED** [review]

**Pull request merged**